collectors/30/kafka_offset_collector.py

```diff
@@ -41,8 +44,31 @@ def format_kafka_consumer_offset_tsd_key(consumer_group, topic, partition, offse
 
 
 def get_partitions(zk, group, topic):
-    return zk.get_children(KafkaPaths.topic_offsets(group, topic))
-
+    try:
+        return map(int, zk.get_children(KafkaPaths.topic_offsets(group, topic)))
+    except NoNodeError as err:
+        logging.exception(err)
+        return []
+
+"""
+Return a list of all kafka topics, or None if the topics cannot be fetched
```

Comment:
Also it's in the wrong place; it should be the first line(s) of the function, not above it.